systems-management/nisysmgmt.yml

```diff
@@ -0,0 +1,2030 @@
+    post:
+      operationId: GetSystemsReport
+      x-ni-privilege: Read
+      x-ni-operation: getSystems
+      tags: [systems]
+      summary: Get systems report
+      description: Get a report containing software/hardware information about the requested systems.
+      parameters:
+        - in: body
+          name: SystemsReportRequest
+          description: The type of the request followed by the list of system Ids.
+          required: true
+          schema:
+            $ref: '#/definitions/SystemsReportRequest'
+      produces:
+        - text/csv
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            description: The file data
+            type: file
+          examples:
+            application/octet-stream:
+              File data
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/create-groups:
+    post:
+      operationId: CreateGroups
+      x-ni-privilege: Write
+      x-ni-operation: createOrUpdateGroups
+      tags: [groups]
+      summary: Create groups
+      description: Create groups.
+      parameters:
+        - in: body
+          name: CreateGroupRequest
+          description: Request for creating groups.
+          required: true
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/CreateGroupRequest'
+      responses:
+        200:
+          $ref: '#/responses/PartialSuccess'
+        204:
+          description: No Content
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+    get:
+      operationId: GetGroups
+      x-ni-privilege: Read
+      x-ni-operation: getGroups
+      tags: [groups]
+      summary: Get groups
+      description: Get groups information.
+      parameters:
+        - in: query
+          name: gid
+          type: string
+          description: A group ID.
+      responses:
+        200:
+          description: OK
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/GroupInfo'
+        400:
+          $ref: '#/responses/BadRequest'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/remove-groups:
+    post:
+      operationId: RemoveGroups
+      x-ni-privilege: Write
+      x-ni-operation: createOrUpdateGroups
+      tags: [groups]
+      summary: Remove groups
+      description: Remove groups.
+      parameters:
+        - in: body
+          name: Groups
+          description: Array of strings representing group ids.
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        200:
+          $ref: '#/responses/Part
```

Comment:
Should this route be just /v1/query-jobs?  That seems to be the preferred style in other apis.